homeassistant/components/iron_os/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration does not have actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: todo
```

Comment:
maybe add in the comment that it's about the data descriptions

Reply:
not sure what to put here as data_description 🤔
![image](https://github.com/user-attachments/assets/ee0ba83e-c889-47a1-8ec5-556b3a27cdd7)